govcd/cse/tkg_versions.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
Users won't need this map (unless they want to check for curiosity), as the map key is retrieved from the input TKG OVA that the user sets as part of cluster creation settings, automatically. Then, with the Key we retrieve the subcomponents versions (etcd, coreDNS, etc).

The users in this case only need to provide a correct TKG OVA (otherwise they get an error on creation)